IBAnimatable/PresentationKeyboardTranslation.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Created by Tom Baranes on 13/08/16.
+//  Copyright © 2016 IBAnimatable. All rights reserved.
+//
+
+import UIKit
+
+public enum ModalKeyboardTranslation {
+  case None
+  case MoveUp  
+  case AboveKeyboard
+
+  static public func fromString(string: String?) -> ModalKeyboardTranslation {
+    if string == "MoveUp" {
```

Comment:
Yeah I think of this, but since all the enums in master are case-insensitive, I prefer to let it that way since it will be fixed in swift3 with the new implementation :)